src/invocation/InvocationService.ts

```diff
@@ -132,6 +163,73 @@ export class Invocation {
 
         return false;
     }
+
+    notify(clientMessage: ClientMessage): void {
+        assert(clientMessage != null, 'Response can not be null');
+        const expectedBackups = clientMessage.getNumberOfBackupAcks();
+        if (expectedBackups > this.backupsAcksReceived) {
+            this.pendingResponseReceivedMillis = Date.now();
```

Comment:
This block is executed once per the invocation, right?

Reply:
Yes, it's executed once per response from the target member.